lib/signs/bus.ex

```diff
@@ -133,7 +182,14 @@ defmodule Signs.Bus do
     Process.send_after(pid, :run_loop, 1_000)
   end
 
-  defp display_route(%{route_id: "746"}), do: "SLW"
+  # Don't display "SLW" route name when its outbound headsign already says "Silver Line Way".
+  defp display_route(%{route_id: "746", headsign: "Silver Line Way"}), do: nil
```

Comment:
It depends on more than just the route id, so what about e.g. `prediction_route_name`?

Also, I think this could be a good candidate for moving out into one of the utility modules.

Reply:
ah yeah that is true. `prediction_route_name` sounds good to me! And agreed on the second point as well.